lib/index.ts

```diff
@@ -3,6 +3,7 @@
 
 import {launch, LaunchedChrome} from 'chrome-launcher';
 const lighthouse = require('lighthouse');
+const parseChromeFlags = require('lighthouse/lighthouse-cli/run').parseChromeFlags;
```

Comment:
This parseChromeFlags method documentation says it is available an as export for testing purposes, see:
https://github.com/GoogleChrome/lighthouse/blob/a2bcf82c13e18a5591e582e6f5760455783b9bb3/lighthouse-cli/run.js#L27

Is Lighthouse using another method to pass on flags to chrome or this it just a documentation typo?
If the former then this change may work for now, but it won't be future-proof

Reply:
Yeah, just wanted to avoid copy/pasta since LH and pwmetrics tightly coupled

>Is Lighthouse using another method to pass on flags to chrome or this it just a documentation typo?

I'm not sure.

For preventing problems in future I see this scenario:
1, of course, add test for this case and use it as it is
2. fork LH and move this method to utilities for safe using

Reply:
I see 3 options with regards to this:
1.) Either use the chrome flags as LH does, as a simple string and pass that over to chrome launcher directly.
2.) Pull Request LH to make that particular function part of the API (would work but I think it's ugly and breaks separation of concerns).
3.) Pull request chrome launcher and address the issue there. Perhaps enabling chrome launcher to take an array of strings.

Reply:
>Either use the chrome flags as LH does, as a simple string and pass that over to chrome launcher directly.

@pedro93 bug was exactly about that. right now doing like `pwmetrics https://example.com --chrome-flags="--headless --no-sandbox --disable-setuid-sandbox"` didn't pass expected params to chrome launcher. Without parsing `chromeFlags` will be passed as string `--headless --no-sandbox --disable-setuid-sandbox` to ChromeLauncher.
Since we are using LH and run ChromeLauncher manually we have to do the same chrome flags adaptation as LH does. 

Sorry that I didn't explain that in a PR description

> Pull request chrome launcher and address the issue there. Perhaps enabling chrome launcher to take an array of strings.

It does that https://github.com/GoogleChrome/chrome-launcher/blob/v0.10.2/src/chrome-launcher.ts already

Reply:
Sorry I didn't reply sooner, I now understand the issue. I think this is ok for now but we must be aware that it will be a hidden dependency on a utility LH method.

Reply:
Yah seems fine. Even if it changes in the future, it will be exposed. So, 👍